crates/jstzd/src/task/octez_client.rs

```diff
@@ -1,8 +1,24 @@
 use super::{endpoint::Endpoint, octez_node::DEFAULT_RPC_ENDPOINT};
-use anyhow::{bail, Result};
+use anyhow::{anyhow, bail, Result};
 use http::Uri;
-use std::{path::PathBuf, str::FromStr};
+use std::{
+    path::{Path, PathBuf},
+    str::FromStr,
+};
 use tempfile::{tempdir, TempDir};
+use tokio::process::Command;
+
+async fn run_command(command: &mut Command) -> Result<()> {
```

Comment:
You can make `run_command` a separate PR. It's actually https://linear.app/tezos/issue/JSTZ-131/implement-octezclientcommand

Reply:
done!  #582